Lib/asyncio/taskgroups.py

```diff
@@ -124,14 +122,17 @@ async def __aexit__(self, et, exc, tb):
             self._errors.append(exc)
 
         if self._errors:
-            # Exceptions are heavy objects that can have object
-            # cycles (bad for GC); let's not keep a reference to
-            # a bunch of them.
-            errors = self._errors
-            self._errors = None
-
-            me = BaseExceptionGroup('unhandled errors in a TaskGroup', errors)
-            raise me from None
+            try:
+                raise BaseExceptionGroup('unhandled errors in a TaskGroup', self._errors) from None
+            finally:
+                # Exceptions are heavy objects that can have object
+                # cycles (bad for GC); let's not keep a reference to
+                # a bunch of them.
+                self._errors = None
```

Comment:
could do this so that `info.append(f'errors={len(self._errors)}')` works ?

```suggestion
                self._errors = range(len(self._errors))
```